prijateli_tree/app/routers/administration.py

```diff
@@ -80,7 +81,10 @@ def confirm_login(
         .one_or_none()
     )
     if user is None:
-        raise InvalidCredentialsException
+        return templates.TemplateResponse(
+            "admin_login.html",
+            {"request": request, "error": "Please submit valid credentials."},
+        )
```

Comment:
<img width="729" alt="Screenshot 2023-11-14 at 9 25 21 PM" src="https://github.com/prijatelilab/PrijateliTree/assets/5885605/e3cfbdd3-dc79-45a4-90cf-c174cbf0897e">

Fixed the redirect for bad login information.